driver/plot_test_outputs.py

```diff
@@ -120,10 +120,14 @@ def collapse_dimensions_for_plotting(longitude, latitude, vname, vx, vd, dims):
             yticks = y
             ylabel = 'Liquid particle size (micron)'
         if vd['yaxis_type'] == 'levStat':
-            y = 480*np.arange(41)
+            y = np.arange(vx.shape[0]+1)
+            if vx.shape[0] == 40:
```

Comment:
Seems like this might come back to bite you. Is there a more robust way to determine if you're using model levels?

Reply:
Indeed, it wasn't the most elegant solution. Information on whether model levels or an evenly-spaced is used is not directly available in the output file. However, when model levels are used levStat is set to zeroes. I've now used this to improve the logic (only a bit more elegant, and more robust).